src/controller/java/src/chip/devicecontroller/model/ClusterState.java

```diff
@@ -79,8 +79,7 @@ public String toString() {
           builder.append("Attribute ");
           builder.append(attributeId);
           builder.append(": ");
-          builder.append(
-              attributeState.getValue() == null ? "null" : attributeState.getValue().toString());
+          builder.append(attributeState.getJson().toString());
```

Comment:
Could the json value be null? It is assumed to be null when a JsonException is thrown.

Reply:
done, thanks